### DISCUSSION TITLE
Where request a wanted feature?

### DISCUSSION THREAD

**Discussion by The-Lum** [plantuml/plantuml#655] [Ideas]
Hi all,

- Where request a wanted feature?

| _First hints, on:_ |
| -- |
| - [forum](https://forum.plantuml.net/wanted-features)
| - [issues](https://github.com/plantuml/plantuml/issues)
| - or perhaps, [discussions](https://github.com/plantuml/plantuml/discussions)...

@arnaudroques: _If you want_, could you add a new category on the [discussions pages](https://github.com/plantuml/plantuml/discussions)?

Regards.

**Comment by VladimirAlexiev** [comment]
Oh, github doesn't add back-links to discussions? @The-Lum see https://github.com/plantuml/plantuml/discussions/1371
